### PR TITLE
NOTICK - Add missing p2p topics and update configuration for compacted topics

### PR DESCRIPTION
**Pull request by dimosr**
Adding some p2p topics that were missing and adding the appropriate configuration to the topics that are supposed to be compacted.

### CODE REVIEW

File: data/topic-schema/src/main/kotlin/net/corda/schema/Schemas.kt [hunk 1]
```diff
@@ -118,13 +118,21 @@ class Schemas {
         companion object {
             const val P2P_OUT_TOPIC = "p2p.out"
             const val P2P_OUT_MARKERS = "p2p.out.markers"
+            const val P2P_OUT_MARKERS_STATE = "p2p.out.markers.state"
```
Comment (yift-r3) [inline data/topic-schema/src/main/kotlin/net/corda/schema/Schemas.kt]: I think we need to add `dlq` as well

(https://github.com/corda/corda-api/pull/408/files)

Comment (dimosr) [reply data/topic-schema/src/main/kotlin/net/corda/schema/Schemas.kt]: good point - let me add that

**Review by yift-r3 — APPROVED** [review]
LGTM

**Comment by dimosr** [comment]
@yift-r3 thanks for reviewing - I will hold off merging this one [until this one](https://github.com/corda/corda-runtime-os/pull/1433) has been approved so that they can both be merged in quick succession.

**Pull request merged**